### DISCUSSION TITLE
Pomaders command

### DISCUSSION THREAD

**Discussion by pincoide** [Jukkales/NecroLens#15] [Ideas]
Would it be possible to make the use of pomaders in a command? Something like "/pom witching"

I was thinking about something like making an "oh crap" button with witching for example 😄

**Comment by Jukkales** [comment]
Thats the hardest part i was thinking off too. Pomander usages are freely not reverse engineered yet. 
I tried myself but have to learn that first :/

**Comment by Jukkales** [comment]
Dont know if youre still interested in, but i just added the command and will push ist soon :D

**Reply by pincoide** [reply]
great, thanks :D
